module/documents/item.mjs

```diff
@@ -2005,6 +2030,42 @@ export default class Item5e extends SystemDocumentMixin(Item) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle summoning from a chat card.
+   * @param {ChatMessage5e} message  The message that was clicked.
+   * @param {Item5e} item            The item from which to summon.
+   */
+  static async _onChatCardSummon(message, item) {
+    let summonsProfile = message.getFlag("dnd5e", "use.summonsProfile");
```

Comment:
We should consider perhaps not storing the chosen summon profile on the card. I imagine one of the use-cases for clicking the chat card summon button will be that you chose the wrong summon, or want to make a different choice, and it won't be possible to do so with this design, if I understand it correctly.